README.md

```diff
@@ -0,0 +1,72 @@
+# Kiwix Apple Custom Apps
+
+Kiwix Apple custom apps are iOS/macOS apps running [Kiwix for
+Apple](https://github.com/kiwix/apple) against a
+pre-configured ZIM file.
+
+This project contains data and scripts needed to create specific
+ custom Kiwix Apple apps.
+
+[![CodeFactor](https://www.codefactor.io/repository/github/kiwix/kiwix-apple-custom/badge)](https://www.codefactor.io/repository/github/kiwix/kiwix-apple-custom)
+[![CI Build Status](https://github.com/kiwix/kiwix-apple-custom/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/kiwix/apple/actions/workflows/ci.yml?query=branch%3Amain)
+[![CD Build Status](https://github.com/kiwix/kiwix-apple-custom/actions/workflows/cd.yml/badge.svg?branch=main)](https://github.com/kiwix/kiwix-apple-custom/actions/workflows/cd?query=branch%3Amain)
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+
+## Custom app folder
+
+[In the repository](https://github.com/kiwix/kiwix-apple-custom),
+each custom configuration is isolated in a so called custom app
+folder. If you need to create a new one for a new custom app, please use a folder name
+that is lowercased and contains no space.
+
+## info.json file
+
+The configuration of the custom app is handled using the `info.json`
+file which **is required to be placed in the custom app folder**. Take example on an already
+existing one if you need to create a new custom app. The most
+important fields are:
+- `about_app_url` - this is an external link that is placed in the "About section" of the application. (Eg. "https://www.dwds.de")
+- `about_text` - this is a custom text that is placed in the "About section" describing what the application is about.
```

Comment:
What is the accepted formatting? Can this contain HTML tags? Newlines ?

Reply:
Good point, I've updated it.